src/mixins/embed.js

```diff
@@ -244,12 +309,61 @@ export default {
             const { total } = totalData;
             this.totalLike = total;
           }),
+          this.updateSuperLikeStatus(),
         ]);
       } catch (err) {
         console.error(err); // eslint-disable-line no-console
       }
     },
-
+    async toggleBookmark() {
+      if (this.isLoadingBookmark) return;
+      this.isLoadingBookmark = true;
+      if (this.bookmarkID) {
+        this.hasBookmarked = true;
```

Comment:
Its seems strange that you mark this flag as true right before deleting bookmark